docs/2.drivers/redis.md

```diff
@@ -72,7 +72,7 @@ const storage = createStorage({
 - `cluster`: List of redis nodes to use for cluster mode. Takes precedence over `url` and `host` options.
 - `clusterOptions`: Options to use for cluster mode.
 - `ttl`: Default TTL for all items in **seconds**.
-- `useRaw`: If enabled, `getItemRaw` and `setItemRaw` will use binary data instead of base64 encoded strings. (this option will be enabled by default in the next major version.)
+- `raw`: If enabled, `getItemRaw` and `setItemRaw` will use binary data instead of base64 encoded strings. (this option will be enabled by default in the next major version.)
```

Comment:
I would suggest something like `binary`/`useBinary` here. I think it makes the effect a bit more clear because it changes how raw data is stored, but does not enable/disable the ability to save raw data. Or it could be inverted to `base64` so in the future the default value can be false.

Reply:
I agree that in general `binary` could be more describing than `raw`. In the context of unstorage, we call binary-compatible feature `raw` which was the reason I was thinking to use this. 

Considering it is for short-term solution, are you happy we go with this?

(Also check fb2977a9855bad1637798d87eb82ad850bc87e70, I have extended normalization)

Reply:
Extending normalization to additional variants of TypedArray makes sense, but it should only allow for binary data and continue to throw when other data is passed. Otherwise it would lead to some unexpected behavior as you save an object but get a Buffer back. 

That's also why I think the different naming is helpful. `binary` means that `setItemRaw` accepts binary and `getItemRaw` returns binary. Whereas the existing `raw` behavior accepts binary and other things.

Reply:
This is consistent with all other drivers (both those support binary and those not) -- happy to change it in the future bu† normally, users should use same `key` either for raw or non raw purposes.

Reply:
Hm would it be possible to add an additional option in that case? Either on the driver or the method? What I'm looking for is a way to save binary data and ensure it is binary and saved as binary and when getting, that I get a Buffer in return. 

My assumption is that is how `getItem(key, { type: 'bytes'})`/`setItem(key, value, { type: 'bytes'})` would work in the future.

Reply:
Yes exactly for `type: 'bytes'` we can get strict.

Currently, if someone passes a string to `setItemRaw`, `getItemRaw` gives a `Buffer` representation of `Utf8` bytes which isn't that odd ,is it?

Reply:
Yeah that's what I find a bit unexpected 😅 

Do you think there would be an issue if we added `type: 'bytes'` as a method option here? It would be just for the redis driver at the moment, but done in a forward compatible manner.

Or is it better to use something else and save that wording for later